### PR TITLE
done setting up heroku

### LINKED ISSUE #2: Setup Heroku

### PR DESCRIPTION
**Pull request by xeon-zolt**
https://melix-generator.herokuapp.com/
@meets2tarun i have only added index will add 404 too 
now we can start work on script 
to resolve #2

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Flask
```
Comment (meets2tarun) [inline requirements.txt]: we can also do it like
Flask==0.10.1
gunicorn==19.1.0

Comment (meets2tarun) [inline requirements.txt]: https://github.com/xeon-zolt/meilix-generator/compare/c5bb81cb...49703a9c
The reason behind removing these lines of equality?

Comment (xeon-zolt) [inline requirements.txt]: i was not sure if they are latest and stable 
but removing those will only install latest and stable

**Comment by xeon-zolt** [comment]
@meets2tarun squash your commits its easy to track

**Pull request merged**